src/lib/analytics/clients/tracks.js

```diff
@@ -85,6 +85,10 @@ export default class Tracks implements AnalyticsClient {
 	}
 
 	send( extraParams: {} ): Promise<Response> {
+		if ( process.env.DO_NOT_TRACK ) {
+			return Promise.resolve( 'tracks disabled per DO_NOT_TRACK variable' );
```

Comment:
We're no longer returning the correct type as indicated by the flow type for the function (`Promise<Response>`). Should we update that?